git/help.md

```diff
@@ -31,6 +31,27 @@ The connection configuration accepts the following parameters:
 
 ### Actions
 
+#### Get File
+
+This action is used to get a file from the repository.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|file_path|string|None|True|Path of the new file (inside the repository)|None|
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|file|file|True|File content|
+
+Example output:
+
+```
```

Comment:
Can you get an example of the output?

Reply:
We can pull this from the JSON run output